templates/flavors/base/cluster-template.yaml

```diff
@@ -118,3 +118,6 @@ spec:
           diskSizeGB: 256
           lun: 0
       sshPublicKey: ${AZURE_SSH_PUBLIC_KEY_B64:=""}
+      identity: UserAssigned
+      userAssignedIdentities:
+      - providerID: /subscriptions/${AZURE_SUBSCRIPTION_ID}/resourceGroups/${CI_RG:=capz-ci}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/${USER_IDENTITY:=cloud-provider-user-identity}
```

Comment:
Will this use the newly created one in the scripts above? I think This means that the user identity is the will be used for capz-manager and ASO on the management cluster and the cloudprovider on the workload cluster.  

In a CI, environment this is probably fine but would it be better have these as distinct identities?  This would allow scoping of the permisions on the cloudprovider identity to only do things like create load balancers.  Again, probably fine for our use case but thought I would mention.

Reply:
Yes, this will be using the ephemeral identity created above. I think it would be our general recommendation to use a separate identity here like you mention, but I think we talked about this being ok for CI.